plugins/dubbo/src/main/java/com/megaease/easeagent/plugin/dubbo/AlibabaDubboCtxUtils.java

```diff
@@ -0,0 +1,231 @@
+			RequestContext requestContext = context.serverReceive(alibabaDubboServerRequest);
+
+			Span span = requestContext.span().start();
+			span.kind(alibabaDubboServerRequest.kind());
+			span.name(alibabaDubboServerRequest.name());
+			span.remoteServiceName(ConfigConst.Namespace.DUBBO);
+			span.remoteIpAndPort(rpcContext.getRemoteHost(), rpcContext.getRemotePort());
+			span.tag(DubboTags.SERVER_APPLICATION.name, applicationName);
+
+			context.put(SERVICE_REQUEST_CONTEXT, requestContext);
+		}
+	}
+
+	public static void finishSpan(Context context, MethodInfo methodInfo, DubboTraceConfig config) {
+		Result result = (Result) methodInfo.getRetValue();
+		Invoker<?> invoker = (Invoker<?>) methodInfo.getArgs()[0];
+		Invocation invocation = (Invocation) methodInfo.getArgs()[1];
+		Throwable throwable = methodInfo.getThrowable();
+
+		boolean isConsumer = invoker.getUrl().getParameter(SIDE_KEY, PROVIDER_SIDE).equals(CONSUMER_SIDE);
+		boolean isAsync = RpcUtils.isAsync(invoker.getUrl(), invocation);
+		if (isAsync) {
+			Future<?> f = RpcContext.getContext().getFuture();
+			if (f instanceof FutureAdapter) {
+				ResponseFuture future = ((FutureAdapter<?>) f).getFuture();
+				future.setCallback(new ResponseCallback() {
+					@Override
+					public void done(Object response) {
+						if (response == null) {
+							doFinishSpan(isConsumer, config, context, null, new IllegalStateException("invalid result value : null, expected " + Result.class.getName()));
+							return;
+						}
+						if (!(response instanceof Result)) {
+							doFinishSpan(isConsumer, config, context, null, new IllegalStateException("invalid result type :" + response.getClass() + ", expected " + Result.class.getName()));
+							return;
+						}
+						Result result = (Result) response;
+						doFinishSpan(isConsumer, config, context, result, null);
+					}
+
+					@Override
+					public void caught(Throwable exception) {
+						doFinishSpan(isConsumer, config, context, null, exception);
+					}
+				});
+				return;
+			}
+		}
+		doFinishSpan(isConsumer, config, context, result, throwable);
+	}
+
+	private static void doFinishSpan(boolean isConsumer, DubboTraceConfig config, Context context, Result result, Throwable throwable) {
+		RequestContext requestContext;
+		if (isConsumer) {
+			requestContext = context.remove(CLIENT_REQUEST_CONTEXT);
+		} else {
+			requestContext = context.remove(SERVICE_REQUEST_CONTEXT);
+		}
+
+		if (requestContext == null) {
+			return;
+		}
+
+		try {
+			Span span = requestContext.span();
+			if (span == null || span.isNoop()) {
+				return;
+			}
+			if (throwable != null) {
+				span.error(throwable);
+			}
+			if (result != null) {
+				if (result.getException() != null) {
+					span.error(result.getException());
+				} else {
+					span.tag(RESULT.name, config.argsCollectEnabled() ? JsonUtil.toJson(result.getValue()) : null);
+				}
+			}
```

Comment:
Please add the enabled configuration of the result tag, like `args.collect.enabled`, and the default is `false`.  
In many cases, the return value of Dubbo will be very large, which will affect the performance of the entire application